gradle-conjure/src/test/groovy/com/palantir/gradle/conjure/ConjurePluginTest.groovy

```diff
@@ -536,6 +536,35 @@ class ConjurePluginTest extends IntegrationSpec {
         result.success
     }
 
+    def 'supports generic generators'() {
+        addSubproject(':api:api-postman')
+        file('versions.props') << """
+        com.palantir.conjure.postman:conjure-postman = 0.1.0
+        """.stripIndent()
+        file('api/build.gradle') << """
+        dependencies {
+            conjureGenerators 'com.palantir.conjure.postman:conjure-postman'
```

Comment:
could we make this `conjureGenerator` singular, instead of plural?

Reply:
Agree we should have done that originally. Unfortunately, It would be a break to rename the configuration in `com.palantir.conjure-local` and I think I'd prefer for the configurations to consistently named